test_configs/sockets/quick.test

```diff
@@ -70,7 +70,7 @@
 {
 	prov_name: sockets,
 	test_type: [
-		FT_TEST_LATENCY,
+		FT_TEST_LATENCY
```

Comment:
I noticed that the config files were missing some commas so I added some in but then realized that they were working so then I stopped/tried to change them to what they were for consistency but I guess I stopped in the middle. All of the other config files are missing random commas. Do you want me to go through them and add all of them in?